tests/suites/main_test.function

```diff
@@ -395,7 +405,50 @@ int main(int argc, const char *argv[])
             if( unmet_dep_count == 0 )
             {
                 test_errors = 0;
+
+#if defined(__unix__) || (defined(__APPLE__) && defined(__MACH__))
+                /* Suppress all output from the library unless we're verbose
+                 * mode
+                 */
+                if( !option_verbose )
+                {
```

Comment:
I did consider that but felt the whole function needed refactoring at a later date. We could do this now however.